plugin/pkg/scheduler/algorithm/priorities/util/topologies.go

```diff
@@ -52,8 +52,12 @@ func PodMatchesTermsNamespaceAndSelector(pod *v1.Pod, affinityPod *v1.Pod, term
 	return true, nil
 }
 
-// nodesHaveSameTopologyKeyInternal checks if nodeA and nodeB have same label value with given topologyKey as label key.
-func nodesHaveSameTopologyKeyInternal(nodeA, nodeB *v1.Node, topologyKey string) bool {
+// NodesHaveSameTopologyKey checks if nodeA and nodeB have same label value with given topologyKey as label key.
+// Returns false if topologyKey is empty.
+func NodesHaveSameTopologyKey(nodeA, nodeB *v1.Node, topologyKey string) bool {
+	if len(topologyKey) == 0 {
```

Comment:
@davidopp 

I'm probably fine with this change, but this is a semantic change in my opinion. Before this PR, a predicate/priority without speified topology key was correct. With this PR in, this is no longer the case.
Do we really want it? @davidopp for feedback